### PR TITLE
NFDIV-1029 Add response pdf to hub page

### PR DESCRIPTION
**Pull request by lara-mpt**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-1029

### Change description ###
Add response pdf to hub page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/modules/document-download/index.ts [hunk 1]
```diff
@@ -9,26 +9,47 @@ const proxy = require('express-http-proxy');
...
 export class DocumentDownloadMiddleware {
   public enableFor(app: Application): void {
-    const addToReqPath = (req: AppRequest) => {
+    const addApplicationToReqPath = (req: AppRequest) => {
       return req.session.userCase?.documentsGenerated?.find(doc => doc.value.documentType === DocumentType.APPLICATION)
         ?.value.documentLink.document_binary_url;
     };
 
+    const addRespondentAnswersToReqPath = (req: AppRequest) => {
+      return req.session.userCase?.documentsGenerated?.find(
```
Comment (linusnorton) [inline src/main/modules/document-download/index.ts:18]: do you need the ? on req.session.userCase?

File: src/main/steps/applicant1/hub-page/right-column/content.ts [hunk 2]
```diff
@@ -38,7 +43,10 @@ const languages = {
 };
 
 export const generateContent: TranslationFn = content => {
+  const statesWithoutRespondentAnswers = [State.AwaitingAos, State.AosDrafted, State.AosOverdue];
+  const aosSubmitted = !statesWithoutRespondentAnswers.includes(<State>content.formState?.state);
```
Comment (linusnorton) [inline src/main/steps/applicant1/hub-page/right-column/content.ts:47]: I think we  also need a joint application check here. Only sole applications have the respondent answers

Comment (linusnorton) [reply src/main/steps/applicant1/hub-page/right-column/content.ts:47]: You could also just check to see if the document exists in the list

File: src/main/modules/document-download/index.ts (hunk 1 above)
Comment (lara-mpt) [reply src/main/modules/document-download/index.ts:18]: nope, I'll remove it

**Review by velimd — APPROVED** [review]

**Pull request merged**